src/tbbbind/tbb_bind.cpp

```diff
@@ -191,6 +191,40 @@ class platform_topology {
                 }
             }
         }
+        // On hybrid CPUs, check if there are cores without L3 cache.
+        if (!core_types_parsing_broken && core_types_number > 1) {
+            // The first core type mask (least performant cores)
+            hwloc_cpuset_t& front = core_types_affinity_masks_list.front();
+            hwloc_cpuset_t lp_mask = hwloc_bitmap_dup(front);
```

Comment:
Could you please add this logic to `common_arena_constraints.h` as well to better test it?

Reply:
Added.

Reply:
Also added `set_core_types` tests.